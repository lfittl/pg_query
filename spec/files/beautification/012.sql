SELECT CASE relkind WHEN 'r' THEN 'TABLE' when 'i' THEN 'INDEX' ELSE 'STH-ELSE' END as type FROM pg_class
