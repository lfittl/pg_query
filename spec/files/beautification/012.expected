SELECT
  CASE "relkind"
    WHEN 'r' THEN 'TABLE'
    WHEN 'i' THEN 'INDEX'
    ELSE 'STH-ELSE'
  END AS type
FROM
  "pg_class"
